scripts/vet-proto.sh

```diff
@@ -32,7 +32,7 @@ elif [[ "$#" -ne 0 ]]; then
 fi
 
 # - Check that generated proto files are up to date.
-go generate google.golang.org/grpc/... && git status --porcelain 2>&1 | fail_on_output || \
+go generate ./... && git status --porcelain 2>&1 | fail_on_output || \
```

Comment:
I think the complete fix would be to run `go generate` on all directories that contain a `go.mod` file.

Reply:
Placed this in a loop